packages/config-helper/project.json

```diff
@@ -32,14 +32,15 @@
 		"bump-version": {
 			"executor": "@jscutlery/semver:version",
 			"options": {
-				"versionTagPrefix": "${target}@"
+				"versionTagPrefix": "${target}@",
+				"postTargets": ["config-helper:publish-npm"]
```

Comment:
Applyed

Reply:
I didn't know that it was possible!

Where can I read more about those tags on the workspace?

Reply:
This feature with ${target} is implemented in https://github.com/jscutlery/semver itself.
I just looked at the code and it seems not to be implemented for postTargets.

I'll merge the PR and continue from there on - maybe the target 'self' used for dependencies in nx is working in such context.

Reply:
In case you are interested in the code in semver,

here is the line that parses a single postTarget string: https://github.com/jscutlery/semver/blob/main/packages/semver/src/executors/version/utils/post-target.ts#L25

and here is the function that replaces the `${...}` vars in the props: https://github.com/jscutlery/semver/blob/main/packages/semver/src/executors/version/utils/resolve-interpolation.ts